package com.part_of_fa.usuarios_documentos.utils.exceptions;

public class InvalidPrestamoException extends RuntimeException {
    public InvalidPrestamoException(String message) {
        super(message);
    }
}